fiftyone/server/routes/sample.py

```
@@ -446,75 +444,144 @@ class SampleField(HTTPEndpoint):
+                generated_sample = generated_dataset[generated_sample_id]
+                sync_to_generated_dataset(
+                    generated_dataset, generated_sample, path, field_id, data
+                )
+                if not is_delete:
+                    generated_sample.reload()
+                    return utils.json.JSONResponse(
+                        utils.json.serialize(generated_sample),
+                        headers={"ETag": etag},
+                    )
+            except Exception as err:
+                logger.error(
```

Ruff (0.14.11)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)